e2e/pod.go

```diff
@@ -414,3 +414,30 @@ func recreateCSIPods(f *framework.Framework, podLabels, daemonsetName, deploymen
 
 	return nil
 }
+
+// validateRWOPPodCreation validates the second pod creation failure scenario with RWOP pvc.
+func validateRWOPPodCreation(
+	f *framework.Framework,
+	pvc *v1.PersistentVolumeClaim,
+	app *v1.Pod,
+	baseAppName string) error {
+	var err error
+	// create one more  app with same PVC
+	name := fmt.Sprintf("%s%d", f.UniqueName, deployTimeout)
+	app.Name = name
+	err = createAppErr(f.ClientSet, app, deployTimeout, errRWOPConflict)
+	if err != nil {
+		return fmt.Errorf("application should not go to running state due to RWOP access mode: %w", err)
+	}
```

Comment:
Add an empty line after the closing `}` from if-statements please. This function reads very difficult.

Reply:
Done.

Reply:
Done.